apps/lenra/lib/lenra/services/kubernetes_api_services.ex

```diff
@@ -0,0 +1,184 @@
+          backoffLimit: 0,
+          template: %{
+            metadata: %{
+              name: build_name
+            },
+            spec: %{
+              initContainers: [%{
+                name: "get-lenra",
+                image: "alpine/curl",
+                command: ["/bin/sh"],
+                args: ["-c", "/tmp/lenra-scripts/get-lenra.sh"],
+                imagePullPolicy: "IfNotPresent",
+                volumeMounts: [%{
+                  mountPath: "/tmp/lenra",
+                  name: "lenra"
+                }, %{
+                  mountPath: "/tmp/lenra-scripts",
+                  name: "lenra-scripts"
+                }]
+              }, %{
+                name: "get-app",
+                image: "alpine/git",
+                command: ["/bin/sh"],
+                args: ["-c", "/tmp/lenra-scripts/get-app.sh"],
+                envFrom: [%{
+                  secretRef: %{
+                    name: build_name
+                  }
+                }],
+                imagePullPolicy: "IfNotPresent",
+                volumeMounts: [%{
+                  mountPath: "/tmp/app",
+                  name: "app"
+                }, %{
+                  mountPath: "/tmp/lenra-scripts",
+                  name: "lenra-scripts"
+                }]
+              }],
+              containers: [%{
+                name: "build",
+                image: "docker:23.0.1-dind-rootless",
+                args: ["/tmp/lenra-scripts/build.sh"],
+                envFrom: [%{
+                  secretRef: %{
+                    name: kubernetes_build_secret,
+                  },
+                }, %{
+                  secretRef: %{
+                    name: build_name
+                  },
+                }],
+                securityContext: %{
+                  privileged: true
+                },
+                imagePullPolicy: "IfNotPresent",
+                volumeMounts: [%{
+                  mountPath: "/tmp/app",
+                  name: "app"
+                }, %{
+                  mountPath: "/tmp/lenra",
+                  name: "lenra"
+                }, %{
+                  mountPath: "/tmp/lenra-scripts",
+                  name: "lenra-scripts"
+                }],
+                workingDir: "/tmp/app/"
+              }],
+              restartPolicy: "Never",
+              volumes: [%{
+                name: "lenra-scripts",
+                configMap: %{
+                  name: kubernetes_build_scripts,
+                  defaultMode: 365 # 0555
+                }
+              }, %{
+                name: "app",
+                emptyDir: %{}
+              }, %{
+                name: "lenra",
+                emptyDir: %{}
+              }]
+            },
+            resources: %{
+              "limits": %{
+                "cpu":"500m",
+                "memory":"1024Mi"
+              },
+              "requests": %{
+                "cpu":"50m",
+                "memory":"100Mi"
```

Comment:
Missing spaces after `:` that cause compile error
```suggestion
                "cpu": "500m",
                "memory": "1024Mi"
              },
              "requests": %{
                "cpu": "50m",
                "memory": "100Mi"
```